docs/native/src/differences.rst

```diff
@@ -1,3 +1,12 @@
+..  _differences:
+
+Differences between |Office iOS| and |wac|
+==========================================
+
+Unlike |wac|, partners need to implement all Container and Bootstrapper operations. You can find more information about these operations in the :ref:`WOPI implementation requirements for Office for iOS integration section. <requirements>`
```

Comment:
Fixed this-- I wasn't sure if it was a hard requirement to do ecosystem operations or if you could just use the Container & Bootstrapper instead, but now I've confirmed.